src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -96,15 +96,26 @@ const ApplicationDetails = ({
               className="ins-c-inventory-detail__app-tabs"
               inset={'insetMd'}
             >
-              {activeTabs?.map((item, key) => (
-                <Tab
-                  key={key}
-                  eventKey={item.name}
-                  title={item.title}
-                  tabContentRef={item.tabRef}
-                  {...item}
-                />
-              ))}
+              {activeTabs?.map(
+                (
+                  {
+                    systemProfilePrefetched,
+                    component,
+                    nonEdge,
+                    tabRef,
+                    ...item
```

Comment:
What's the goal of spreading the keys that are not used? For example, you explicitly extract `systemProfilePrefetched` or `component` key, but don't utilise it anywhere further down the code.

Reply:
Nvm just removed the item spread, the props weren't used